packages/contracts-bedrock/contracts/libraries/Predeploys.sol

```diff
@@ -78,4 +78,9 @@ library Predeploys {
      *         L2ToL1MessagePasser contract instead.
      */
     address internal constant LEGACY_MESSAGE_PASSER = 0x4200000000000000000000000000000000000000;
+
+    /**
+     * @notice Address of the ProxyAdmin predeploy.
+     */
+    address internal constant PROXY_ADMIN = 0x4200000000000000000000000000000000000017;
```

Comment:
Should this be a predeploy? Will we also make the ProxyAdmin upgradeable?

Could we just deploy it normally, even before state surgery, and then set the owner of the predeploys to that address?

Reply:
We could do that, we talked about this awhile ago on office hours and agreed to this approach but then it never got merged. I don't really care which approach we take.

The proxy admin predeploy is not upgradable, along with the legacy eth and gov token

Reply:
Right. OK, so worst case of making it a predeploy is that we might replace it eventually and then we just have a legacy predeploy floating around that doesn't do much. 

Not ideal, but also not worth a bunch of rework.